cgi-bin/DW/Hooks/EmbedWhitelist.pm

```diff
@@ -65,6 +65,8 @@ my %host_path_match = (
     "docs.google.com"       => [ qr!^/(document|spreadsheets?|presentation)/!, 1 ],
     
     "books.google.com"      => [ qr!^/ngrams/!, 1 ],
+    
+    "jsfiddle.net"          => [ qr!^/.*/embedded/$!, 1 ],
```

Comment:
Using `.*` in regexps is usually a bad idea, causing the computer to do a lot of extra work to match characters we don't care about.  Just match the end of the path: `qr!/embedded/$!`